src/module/actor/SR5Actor.ts

```diff
@@ -2183,6 +2183,14 @@ export class SR5Actor<SubType extends Actor.ConfiguredSubType = Actor.Configured
         })));
     }
 
+    get icType() {
+        if (this.isType('ic')) {
+            // FIXME cast to type IC but that didn't seem to work
```

Comment:
Yeah... that is one of the moments I told you not to use `get`. so you would need to do is: `icType(this: SR5Actor)`

Reply:
That makes sense. Fixing now